sentry/values.yaml

```diff
@@ -234,6 +234,14 @@ clickhouse:
   enabled: true
   clickhouse:
     imageVersion: "19.16"
+    persistentVolumeClaim:
+      enabled: true
+      dataPersistentVolume: 
+        enabled: true
+        accessModes: 
+        - "ReadWriteOnce"
+        storageClassName: "standard"
```

Comment:
Common practice is to set this as `"default"`. May be better?

Reply:
It's unclear in the k8s docs, but I think it's only when you don't specify "storageClassName" that default is used. Could you confirm?

Reply:
Sorry completely missed this reply... Oops. Basically no "baremetal" cluster comes with a storage class, so blank typically defaults to `default` in my experience... Beyond that though really it's down to the user. I don't think it's a big deal either way.